src/webcomponents/spreadsheet/Spreadsheet.stories.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Spreadsheet from './Spreadsheet';
+
+export default {
+    title: 'Spreadsheet',
+    component:Spreadsheet,
+}
+
+export const TwoData = () => <Spreadsheet languages='{"en": {"budget": "BUDGET", "message2": "MESSAGE2"}, "sk": {"budget": "ROZPOCET", "message2": "hlaska 2"}}' />
+export const ThreeData = () => <Spreadsheet languages='{"en": {"budget": "BUDGET", "message2": "MESSAGE2"}, "sk": {"budget": "ROZPOCET", "message2": "hlaska 2"},"de":{"budget": "affaf", "message2": "asf 2"}}' />
```

Comment:
Please, use correct de (german translations - check the screenshot in the issue related to adding new columns)